test/unit/accord/utils/Gens.java

```diff
@@ -19,26 +19,39 @@
 package accord.utils;
 
 import java.lang.reflect.Array;
+import java.math.BigDecimal;
```

Comment:
a lot of changes to these classes are not *needed* for this patch, but my thinking is to try to make in-sync with accord so the next rebase isn't a pain